instruqt/proxy/03-app-install/assignment.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: app-install
+id: zfxhfqtljsrb
+type: challenge
+title: Kots application install with proxy config
+teaser: Install kots application via proxy server
+notes:
+- type: text
+  contents: |
+    # Install kots application using a proxy server
+tabs:
+- title: Host
+  type: terminal
+  hostname: isolated-host
+- title: Vendor
+  type: website
+  url: https://vendor.replicated.com
+  new_window: true
+- title: KotsAdm
+  type: website
+  url: http://isolated-host.${_SANDBOX_ID}.instruqt.io:8800
+  new_window: true
+difficulty: basic
+timelimit: 1800
+---
+
+💡 Shell
+=========
+
+Install sample application with proxy config in place.
+
+The license file for the application has already been downloaded in the home directory.
+
+Even thought the kurl installer spec included kotsadm which can be see running the application installation can be completed by the command line with the kot install command using the license.
+
+Install using kots cli including license file to upload automatically vs uploading via kotsadm web UI
+```
+kubectl kots install proxy-$INSTRUQT_PARTICIPANT_ID --skip-preflights --license-file ~/license.yaml --shared-password mytestapp --namespace default --no-port-forward
+```
+
```

Comment:
Since you've of the KOTS Admin Console available, I'd recommend doing this in the browser.

Reply:
I'm happy with sticking with the CLI here.  There are so many labs that walk through this already and I think this might be the only example of installing using the CLI which is important to have somewhere IMHO.  The whole "proxy" workflow is a very cli kind of things so felt that it sat well here.  Having the admin console UI available does allow the GUI preferred uses that as an option if they want to take that route and have probably got the knowledge of the steps ahead of running this track so is it really necessary to add the screenshots and workflow yet again here?